### PR TITLE
Add new test for MappingFEField for simplex meshes

### PR DESCRIPTION
**Pull request by peterrum**
With the following result with linear and quadratic mapping:

mapping-fe-fields

FYI @agrayver

### CODE REVIEW

**Comment by peterrum** [comment]
Failing test is unrelated and is fixed in #11573.

File: tests/simplex/mapping_fe_fields_02.cc [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+  FEValues<dim> fe_values(mapping,
+                          fe,
+                          quadrature_formula,
+                          update_values | update_gradients);
+
+  for (const auto &cell : dof_handler.active_cell_iterators())
+    {
+      fe_values.reinit(cell);
+    }
```
Comment (kronbichler) [inline tests/simplex/mapping_fe_fields_02.cc:95]: Is this really needed? `DataOut::build_patches` should also call this function (using `update_values` and `update_quadrature_points` as arguments).

File: tests/simplex/mapping_fe_fields_02.cc [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+}
+
+int
+main()
+{
+  initlog();
+
+  test(1); // linear mapping
+  test(2); // quadratic mapping
+}
```
Comment (kronbichler) [inline tests/simplex/mapping_fe_fields_02.cc:123]: Do we need a newline here?

**Comment by peterrum** [comment]
@kronbichler Done!

**Review by kronbichler — APPROVED** [review]

**Pull request merged**